sessions/js-inputs-and-strings/multiple-inputs_solution/index.html

```diff
@@ -6,19 +6,69 @@
       name="viewport"
       content="width=device-width, initial-scale=1, shrink-to-fit=no"
     />
-    <title>JS Inputs & Strings Exercise 4 - Multiple Inputs: Part 3</title>
+    <title>JS Inputs and Strings: Multiple Inputs</title>
     <link rel="stylesheet" href="./css/styles.css" />
+    <script src="./js/index.js" defer></script>
   </head>
   <body>
+    <!-- ↓ form for the first sub-challenge ↓ -->
     <form class="form">
-      <label for="first-input">First Input</label>
-      <input class="form__input" id="first-input" data-js="first-input" />
-      <label for="second-input">Second Input</label>
-      <input id="second-input" class="form__input" data-js="second-input" />
-      <button class="form__button" type="button" data-js="button">
-        Click!
+      <label for="first-copy-input">First Input</label>
+      <input
+        class="form__input"
+        id="first-copy-input"
+        data-js="first-copy-input"
+      />
+      <label for="second-copy-input">Second Input</label>
+      <input
+        id="second-copy-input"
+        class="form__input"
+        data-js="second-copy-input"
+      />
+      <button class="form__button" type="button" data-js="copy-button">
+        Click to copy the first input value into the second input field
+      </button>
+    </form>
+    <!-- ↓ form for the second sub-challenge ↓ -->
+    <form class="form">
+      <label for="first-copy-uppercase-input">First Input</label>
+      <input
+        class="form__input"
+        id="first-copy-uppercase-input"
+        data-js="first-copy-uppercase-input"
+      />
+      <label for="second-copy-uppercase-input">Second Input</label>
+      <input
+        id="second-copy-uppercase-input"
+        class="form__input"
+        data-js="second-copy-uppercase-input"
+      />
+      <button
+        class="form__button"
+        type="button"
+        data-js="copy-uppercase-button"
+      >
+        Click to copy the first input value into the second input field in
+        uppercase
+      </button>
+    </form>
```

Comment:
what do you think about giving each form an h2 for the challenge part? 
```suggestion
    <form class="form">
      <h2>Part 2: Copy and UpperCase</h2>
      <label for="first-copy-uppercase-input">First Input</label>
      <input
        class="form__input"
        id="first-copy-uppercase-input"
        data-js="first-copy-uppercase-input"
      />
      <label for="second-copy-uppercase-input">Second Input</label>
      <input
        id="second-copy-uppercase-input"
        class="form__input"
        data-js="second-copy-uppercase-input"
      />
      <button
        class="form__button"
        type="button"
        data-js="copy-uppercase-button"
      >
        Click to copy the first input value into the second input field in
        uppercase
      </button>
    </form>
```